storage/src/adb/sync/requests.rs

```diff
@@ -1,21 +1,20 @@
 //! Manages outstanding fetch requests
 
 use crate::adb::sync::engine::IndexedFetchResult;
-use commonware_cryptography::Digest;
 use futures::stream::FuturesUnordered;
 use std::{collections::BTreeSet, future::Future, pin::Pin};
 
 /// Manages outstanding fetch requests
-pub(super) struct Requests<Op, D: Digest, E> {
-    /// Futures that will resolve to batches of operations
+pub(super) struct Requests<D, P, E> {
```

Comment:
`D` is for `Data` here, not `Digest` like usual. Would it be more clear if I made this `Data`?